Software/Python/ir_remote_control/gobox_ir_receiver_libs/ir_debug_logger.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python 
```

Comment:
Purpose for this file? 
Is it just a debugging tool that support can ask clients to try, or is it actively running?